packages/documentation/src/stories/components/stepper/stepper.stories.ts

```diff
@@ -104,10 +113,17 @@ function getStepperItem(args: Args, step: string, index: number, updateStep: (ne
   `;
 }
 
-
 function renderStepper(args: Args) {
   const [_, updateArgs] = useArgs();
 
+  // Hack to force a reflow when the current step changes
+  const stepperEl = document.querySelector('.stepper') as HTMLElement;
+  if (stepperEl) {
+    stepperEl.style.display = 'none';
+    console.log(stepperEl.offsetHeight);
```

Comment:
There are other ways to make the browser flush the queue of styles to apply. Check out `queueMicrotask` or `requestAnimationFrame` (https://stackoverflow.com/questions/23253976/how-to-properly-wait-for-browser-reflow-repaint-to-finish/23254061).